### ISSUE TITLE
Feature request: Mute microphone

### ISSUE THREAD

**Issue by STM3900** [CodinGlitch/SimpleRadio#40]
The microphone is a great feature, but I'm a bit annoyed because there are times when I'd like the microphone not to transmit what I'm saying

My current solution is to break it and put it down just when I need it, but I think it would be a great addition to be able to mute it by doing shift right click for example?

You could put them permanently in specialized places, like a recording studio, or elsewhere!

**Comment by CodinGlitch** [comment]
yeah i believe the original plan i had was to use that big red button on the front for it, but i haven't gotten to implementing it in-game yet in favor of getting the version out sooner

**Comment by CodinGlitch** [comment]
the microphone can now be muted with SR3
